docs/install/index.md

```diff
@@ -94,11 +94,35 @@ Go and explore your InvenioRDM instance on:
 
 To create a new administrator account:
 
+
+**Local application**
+
 - `pipenv run invenio users create <EMAIL> --password <PASSWORD>`
 - Activate it with `pipenv run invenio users activate <EMAIL>`
 - If you have email verification enabled (as per defaults) you may want to [verifiy the account manually](https://inveniordm.docs.cern.ch/customize/vocabularies/users/#confirm-user)
 - Allow the user to access the administration panel: `pipenv run invenio access allow administration-access user <EMAIL>`
 
+**Containerized application**
+
+1. Login to your `web-api-1` container (use `docker ps` to find the exact name): `docker exec -it my-site-web-api-1 bash`
+
+2. Within the `web-api-1` container enter the following commands to create a first user
+
+- Create User: `invenio users create <EMAIL> --password <PASSWORD>`
+- Activate User: `invenio users activate <EMAIL>`
+- Give User Admin Role: `invenio roles add <EMAIL> admin`
+- Confirm Email using `invenio shell` and then typing
+  ```
+  from flask_security.confirmable import confirm_user
+  from invenio_accounts.proxies import current_datastore
+  from invenio_db import db
+  user=current_datastore.get_user("<EMAIL>")
+  confirm_user(user)
+  db.session.commit()
+  reindex_users([user.id])
+  ```
+3. Login with <EMAIL> and <PASSWORD> to https://127.0.0.1/login/
+
```

Comment:
I like the numbered steps and specifying to connect to a container. The 2 approaches are essentially the same, so how about:

```suggestion
Depending on whether you are in a local or containerized setup, take note of the variations immediately following before stepping through the subsequently outlined steps. 

**Local application**

Per usual, in a local application context, precede the `invenio` commands by `pipenv run` (e.g., `pipenv run invenio users create <EMAIL> --password <PASSWORD>`).

**Containerized application**   
In a fully containerized context, connect to a container first e.g. the web-api container: `docker exec -it my-site-web-api-1 /bin/bash`. Then run the commands from within the container as-is. 

**Steps**
1- `invenio users create <EMAIL> --password <PASSWORD>`
2- Activate it with `invenio users activate <EMAIL>`
3- If you have email verification enabled (as per defaults), you may want to [verifiy the account manually](https://inveniordm.docs.cern.ch/customize/vocabularies/users/#confirm-user) using the user's <EMAIL>.
4- Allow the user to access the administration panel: `invenio access allow administration-access user <EMAIL>`
```